dll/cpl/sysdm/performance.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * PROJECT:     ReactOS System Control Panel Applet
+ * LICENSE:     LGPL-2.1+ (https://spdx.org/licenses/LGPL-2.1+)
+ * FILE:        dll/cpl/sysdm/performance.c
+ * PURPOSE:     Performance settings property sheets
+ * COPYRIGHT:   Copyright 2024 Whindmar Saksit <whindsaks@proton.me>
+ *
+ */
+
+#define COBJMACROS
+#include "precomp.h"
```

Comment:
Isn't putting the define before the precompiled header inclusion, modifying its definitions? If so, the precompiled header isn't going to be used since the definitions with which it has been compiled, would become different.
It may be of interest to move the COBJMACROS define inside the precomp.h itself.